frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -1570,7 +1570,7 @@ export const widgets = [
       defaultTab: { type: 'code', displayName: 'Default tab' },
       hideTabs: { type: 'toggle', displayName: 'Hide Tabs' },
     },
-    events: {},
+    events: { onClick: { displayName: 'On click' } },
```

Comment:
```suggestion
    events: { onTabSwitch: { displayName: 'On tab switch' } },
```
`On click` will imply a different kind of meaning @stepinfwd . Let's use `onTabSwitch`.  You will have to add this event handle on the array that starts at `appUtils.js:473` as well as this is a new type of event.